llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -279,7 +336,8 @@ InstructionCost RISCVTTIImpl::getShuffleCost(TTI::ShuffleKind Kind,
             // Example sequence:
             //   vnsrl.wi   v10, v8, 0
             if (equal(DeinterleaveMask, Mask))
-              return LT.first * TLI->getLMULCost(LT.second);
+              return LT.first * getRISCVInstructionCost({RISCV::VNSRL_WI},
```

Comment:
If there is only one element, we can drop the braces.

Reply:
Fixed. Thanks!